constants.js

```diff
@@ -62,6 +63,7 @@ const checksWhitelist = {
     [editEvent]: [],
     [issuesLabelEvent]: []
   },
+  // eslint-disable-next-line quote-props
```

Comment:
I have Disabled Lint beacuse while commiting `eslint *.js ./spec/*.js ./lib/*.js actions/**/*.js --fix` runs causing all auto fixable problems to be fixed.